config/bidderinfo.go

```diff
@@ -246,6 +246,35 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 			infos[string(normalizedBidderName)] = info
 		}
 	}
+	return processAliasBidderInfo(infos)
+}
+
+func processAliasBidderInfo(infos BidderInfos) (BidderInfos, error) {
+	for bidderName, bidderInfo := range infos {
+		if len(bidderInfo.AliasOf) > 0 {
+			if err := validateAliases(bidderInfo, infos, bidderName); err != nil {
+				return nil, err
+			}
+			parentBidderInfo := infos[bidderInfo.AliasOf]
+			bidderInfo.AppSecret = parentBidderInfo.AppSecret
+			bidderInfo.Capabilities = parentBidderInfo.Capabilities
+			bidderInfo.Debug = parentBidderInfo.Debug
+			bidderInfo.Disabled = parentBidderInfo.Disabled
+			bidderInfo.Endpoint = parentBidderInfo.Endpoint
+			bidderInfo.EndpointCompression = parentBidderInfo.EndpointCompression
+			bidderInfo.Experiment = parentBidderInfo.Experiment
+			bidderInfo.ExtraAdapterInfo = parentBidderInfo.ExtraAdapterInfo
+			bidderInfo.GVLVendorID = parentBidderInfo.GVLVendorID
+			bidderInfo.Maintainer = parentBidderInfo.Maintainer
+			bidderInfo.ModifyingVastXmlAllowed = parentBidderInfo.ModifyingVastXmlAllowed
+			bidderInfo.OpenRTB = parentBidderInfo.OpenRTB
+			bidderInfo.PlatformID = parentBidderInfo.PlatformID
+			bidderInfo.Syncer = parentBidderInfo.Syncer
+			bidderInfo.UserSyncURL = parentBidderInfo.UserSyncURL
+			bidderInfo.XAPI = parentBidderInfo.XAPI
+			infos[bidderName] = bidderInfo
```

Comment:
May be we should not inherit config here. Consider scenario where alias adapter is disabled  and parent adapter is not. As per above code, although child adapter is disabled but `bidderInfo.Disabled` will be true.

Also params like `GVLVendorID` shouldn't be inherited. We have `applyBidderInfoConfigOverrides()` which decides which params should be inherited. Should try to use this method
https://github.com/prebid/prebid-server/blob/67643525139934f260787b610eaadee298f61138/config/bidderinfo.go#L435